sirepo/reply.py

```diff
@@ -430,6 +443,14 @@ def _content_type(resp):
             r.set_header("Content-Type", c)
 
         def _cookie(resp):
+            def _delete():
```

Comment:
too many levels of nesting. just called it _cookie_aux_delete

Reply:
https://github.com/radiasoft/sirepo/pull/5523/commits/711e56e4b6d7aae82470e59ce8f2c1e71199f412